packages/react-components/react-portal/src/virtualParent/getParent.ts

```diff
@@ -1,9 +1,34 @@
-import { getVirtualParent } from './getVirtualParent';
+import { isVirtualElement } from './isVirtualElement';
+
+/**
+ * Gets the virtual parent given the child element, if it exists.
+ */
+function getVirtualParent(child: Node): Node | null {
+  return isVirtualElement(child) ? child._virtual.parent || null : null;
+}
```

Comment:
These is no sense to have this function in a separate file, it's also not exported anywhere.